src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/mouse.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'mouse()'
+excerpt: 'Browser module: page.mouse method'
+---
+
+Returns the [Mouse](javascript-api/k6-experimental/browser/mouse/) instance to interact with a virtual mouse on the page.
+
```

Comment:
We should probably have a _Returns_ sub heading in here.

Reply:
Resolve in https://github.com/grafana/k6-docs/commit/95398b184f9afe52df89593693d16dfa2d7085c5